packages/firestore/src/local/query_cache.ts

```diff
@@ -88,6 +100,11 @@ export interface QueryCache extends GarbageSource {
     queryData: QueryData
   ): PersistencePromise<void>;
 
+  /**
+   * The number of targets currently in the cache.
+   */
+  count(): number;
```

Comment:
FWIW, this could be a property since it's synchronous and cheap...

Reply:
Ah, that's neat. I didn't realize typescript could do that.